geospaas_harvesting/harvesters.py

```diff
@@ -17,13 +17,21 @@ class HarvesterConfigurationError(Exception):
     missing mandatory sections
     """
 
+
 class Harvester():
     """
     Base Harvester class. Implements the basic behavior but is not meant to be used directly.
     It should be subclassed, and child classes should implement the following methods:
         - _create_crawlers()
         - _create_ingester()
+
+    Two attributes of ingester and crawler are set here as None (as they are in the parent class).
+    These two may be specified for each harvester in order for that harvestor to be operational.
+    These two attributes of "ingester" and "crawler" may need by definition of "_create_crawlers"
+    and "_create_ingester" methods for creating the diferent kind(s) of harvesters.
```

Comment:
Try to avoid using verbose phrasing in the explanations. The more direct, the better!
```suggestion

    Child classes should also assign values to the ingester and crawler class attributes.
    These values should be the crawler and ingester classes to be used in the _create_crawlers()
    and _create_ingester() methods.
```

Reply:
Done in the next commit